bundle/Core/Pagination/Pagerfanta/RelatedContentAdapter.php

```diff
@@ -54,9 +64,30 @@ public function setTag(Tag $tag)
         $this->tag = $tag;
     }
 
+    /**
+     * @param \eZ\Publish\API\Repository\Values\Content\Query\SortClause[] $sortClauses
+     */
+    public function setSortClauses(array $sortClauses)
+    {
+        $this->sortClauses = $sortClauses;
+    }
+
+    /**
+     * Sets additional criteria to be used in search.
+     *
+     * @param array $additionalCriteria
```

Comment:
Typehint should be `\eZ\Publish\API\Repository\Values\Content\Query\Criterion[]`, right?

Reply:
Fixed in https://github.com/netgen/TagsBundle/pull/98/commits/05f7dad27c86fbfdc247c7f08f2966bc63b3a183.